portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,69 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function randomizeImage() {
+  // The images directory contains 13 images, so generate a random index between
+  // 1 and 13.
+  const imageIndex = Math.floor(Math.random() * 24) + 1;
+  const imgUrl = 'images/jfew-' + imageIndex + '.jpg';
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const imgElement = document.createElement('img');
+  imgElement.src = imgUrl;
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const imageContainer = document.getElementById('random-image-container');
+  // Remove the previous image.
+  imageContainer.innerHTML = '';
+  imageContainer.appendChild(imgElement);
+}
+
+// modalbtn opens the first modal
+function Intromodal() {
+  let button = document.getElementById("Intro");
```

Comment:
sorry if i led you astray earlier, but if the variable isn't being reassigned, you should use `const` instead of `let`

This applies to all your modal functions

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let